Module4_Labs/Lab3_File_System/Cards/111.md

```diff
@@ -3,11 +3,6 @@
 <!--badges={Python:22,Algorithms:11}-->
 
 <!--concepts={directedGraphs, introToGraphs, useOfGraphs}-->
-
-Here is a diagram of the file structure with the root and subfolder, etc. Make sure to study the file structure before moving onto the coding details.
-
-<img src = "https://i.imgur.com/d96KQDv.jpg" width = "450px"/>
-
 Let's get into the implementation.
 
 First, we have to open and read the file line by line in order to read every folder and its associated number:
```

Comment:
I would add this to 11.md

Reply:
I resolved the issue by adding a concluding sentence to transition to the next card. All of the code should only be in the easy card (Card 111) because the purpose of that card is to actually give away the answer.